vitals/BatteryStatusChecker.java

```diff
@@ -0,0 +1,49 @@
+package vitals;
+
+public class BatteryStatusChecker {
+
+	static void printBatteryStatus(String batteryStatus, float abnormalValue) {
+		System.out.println(batteryStatus + " Abnormal Value: " + abnormalValue);
+	}
+
+	static Battery checkIfBatteryTemperatureIsHigh(Battery battery) {
+		if (battery.temperature > 45) {
+			printBatteryStatus("Temperature is high!", battery.temperature);
+			battery.batteryStatus = false;
+		} 
+
+		return battery;
+	}
```

Comment:
why these check********** function return battery object.....
duplicate conditional statement in each of the check*********** function
Applying printBatteryStatus in each of the check************ function makesdifficult to test check*** functions